### PR TITLE
Add New Relic's Helm chart repository

### PR DESCRIPTION
**Pull request by douglascamata**

### CODE REVIEW

**Comment by mattfarina** [comment]
The [Artifact Hub](https://artifacthub.io) is another place you can list your repos. To do that, you login (or create an account), create an organization, and list your repository. You can link it to a user or organization.

File: repos.yaml [hunk 1]
```diff
@@ -1313,3 +1313,10 @@ repositories:
         email: kubernetes-sig-ui@googlegroups.com
       - name: Cédric de Saint Martin
         email: cedric@desaintmartin.fr
+  - name: newrelic
+    url: https://helm-charts.newrelic.com
```
Comment (mattfarina) [inline repos.yaml:1317]: The repository also needs to be added to the `config/repo-values.yaml` file.

Comment (douglascamata) [reply repos.yaml:1317]: Fixed in 4b6fff3, thanks!

**Comment by douglascamata** [comment]
Thanks for suggesting Artifact Hub, @mattfarina! Will definitely consider it too.

**Pull request merged**